ArgoProject/src/components/Player.h

```diff
@@ -0,0 +1,36 @@
+﻿#ifndef _PLAYER_H
+#define _PLAYER_H
+
+
+
+namespace app::comp
+{
+	class Player
+	{
+	public: // Constructors/Destructor/Assignments
+		Player() = default;
+		~Player() = default;
+
+		Player(Player const &) = default;
+		Player & operator=(Player const &) = default;
+
+		Player(Player &&) = default;
+		Player & operator=(Player &&) = default;
+
+	public: // Public Static Functions
+	public: // Public Member Functions
+	public: // Public Static Variables
+	public: // Public Member Variables
+		bool falling = false;
```

Comment:
Why is this variable in a `Player` component?

Reply:
It's for when the player presses down to go through the platform if this bool is true then it ignores the platform collision. (Working on this right now)